.github/workflows/ci.yml

```diff
@@ -22,12 +23,14 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install jina[cicd,test] --no-cache-dir
+          pip install jina[cicd,test,http] requests --no-cache-dir
+          pip install -r ${{ matrix.examples }}/requirements.txt --no-cache-dir
       - name: Test examples with pytest
         run: |
           jina check
           export JINA_LOG_VERBOSITY="ERROR"
-          pytest -s -v */tests
+          cd ${{ matrix.examples }}
+          pytest -s -v .
```

Comment:
Better pass the path to the arguments

Reply:
Do you mean rename `examples` in matrix to `path`?

Reply:
No, I mean call `pytest -v -s ${{matrix.examples}}` instead of doing cd $...

Reply:
We need the `cd` for it to access the pods. I removed all `chdir` from the code and so need to do cd here

Reply:
Yes, I like this solution much more, than how it was before. Great job! :+1: